engine/components-api/src/main/java/com/cloud/vm/snapshot/VMSnapshotManager.java

```diff
@@ -28,14 +28,35 @@
 
 public interface VMSnapshotManager extends VMSnapshotService, Manager {
 
-    static final ConfigKey<Integer> VMSnapshotExpireInterval = new ConfigKey<Integer>("Advanced", Integer.class, "vmsnapshot.expire.interval", "-1",
-            "VM Snapshot expire interval in hours", true, ConfigKey.Scope.Account);
+    Integer defaultVirtualMachineSnapshotsMax = 10;
+    Integer defaultVirtualMachineJobCheckInterval = 3000;
+    Integer defaultVirtualMachineSnapshotExpireInterval = -1;
+    Integer defaultVirtualMachineSnapshotCreateWait = 1800;
 
-    public static final int VMSNAPSHOTMAX = 10;
+    ConfigKey<Integer> virtualMachineSnapshotExpireInterval = new ConfigKey<Integer>("Advanced",
```

Comment:
isn't convention for config-values capatalised?

Reply:
@DaanHoogland Well, there are different kinds of coding conventions I have met in the codebase. Why I use the style like that here? The answer is because those definitions are used only once to define ConfigKey variables and some kind of syntax sugar which improves manageability.
If it should be changed to a capitalized style, can do that.

Reply:
ok, not making a point of it.